### DISCUSSION TITLE
goto destination when only 1 option in database

### DISCUSSION THREAD

**Discussion by Masber** [ajeetdsouza/zoxide#915] [Q&A]
Dear zoxide community,

I am testing zoxide and have a question,

I am a fan of the autocomplete feature where zoxide is able to show a list of directories + score I have been before.
My issue with this feature is that, often, there is only one entry in the database and I have to press <ENTER> twice in order to go to that folder.

- The first enter will add the zoxide command to go to that directory (eg: `$ z my_folder z#/home/my_user/my_folder/`).
- Then I have to press <ENTER> for second time in order to go to the directory.

My question is:

When there is only one entry in zoxide database, is it possible to configure zoxide so it skips the step where it prints the zoxide command on screen and it just takes me to the directory after selecting from the suggestion list?

thank you

**Comment by ajeetdsouza** [comment]
What shell are you using? Also, to clarify, are you saying:

- You don't want the interactive selector to pop up when there's only one option, or
- When you press `<Return>` in the interactive selector, you don't want the `z#` command to show up?

**Comment by Masber** [comment]
I am using Bash

both, I was complaining about the fact of showing the z# command and then press enter but getting rid of the interactive selector when only 1 option available would be even better

**Reply by Masber** [reply]
but even when interactive selector shows multiple entries... what is the point of showing the z# command and for the user to press enter again? if user already made the choice in the interactive selection, then just take the user there.
If you think it is useful to show the user extra information like the z# command, then ask the user to run in debug mode or something

**Reply by Masber** [reply]
should I create an issue for this?
